Editor/Window/ConnectToFleetInput.cs

```diff
@@ -65,7 +65,7 @@ private async Task OnAnywhereConnectClicked(string fleetName)
                 if (!customLocationResponse.Success)
                 {
                     _connectToAnywhereStatusBox.Show(StatusBox.StatusBoxType.Error,
-                        Strings.AnywherePageStatusBoxDefaultErrorText, customLocationResponse.ErrorMessage);
+                        Strings.AnywherePageStatusBoxDefaultFleetErrorText, customLocationResponse.ErrorMessage);
```

Comment:
This isn't entirely accurate, but since the location creation is tied into the fleet creation and not a separate step on ux i'm going to leave this as is